convert/cloudformation-only/main.tf

```diff
@@ -14,7 +14,7 @@ resource "aws_iam_policy" "cloudformation_only" {
 }
 
 resource "aws_iam_role" "cloudformation_only" {
-  name = "cloudformation-only"
+  name = "cloudformation-only${var.suffix}"
```

Comment:
Both of these resources support `name_prefix` if you want TF to handle it

Reply:
Doing it this way means I can also associate it with the deploy name.